centralserver/settings.py

```diff
@@ -182,18 +182,14 @@
     'django.contrib.humanize',
     'centralserver.contact',
     'kalite.updates',
-    'kalite.caching',
     'centralserver.i18n',
     'tastypie',
     'announcements',
-    'kalite.playlist',
```

Comment:
This will delete all of the Nalanda `QuizLog` data. We probably don't want to do that?